### ISSUE TITLE
Deprecated kubernetes api policy/v1beta1

### ISSUE THREAD

**Issue by JeroennC** [jfrog/charts#1618]
**Is this a request for help?**: yes

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Request

**Version of Helm and Kubernetes**: Kubernetes 1.21+

**Which chart**: artifactory-ha

**What happened**:
During installation of the artifactory-ha charts, deprecation warnings trigger:
```
policy/v1beta1 PodDisruptionBudget is deprecated in v1.21+, unavailable in v1.25+; use policy/v1 PodDisruptionBudget
```

**What you expected to happen**:
No warnings, the new api should be used if the kubernetes version is v1.21+

**How to reproduce it** (as minimally and precisely as possible):
Install artifactory-ha with minAvailable configured:
https://github.com/jfrog/charts/blob/master/stable/artifactory-ha/templates/artifactory-primary-pdb.yaml#L2 

**Anything else we need to know**:
There is also a pdb for the node statefulset.

**Comment by rahulsadanandan** [comment]
@JeroennC thanks for reporting this. We would be fixing this in the upcoming releases.

**Comment by chukka** [comment]
This is already fixed internally , will be available in 7.40.x release

**Comment by chukka** [comment]
> This is already added internally , will be available in 7.40.x release

Update : 7.41.x release
